DesignSafe-CI/portal
# Task/des 2335 my projects hide show desc

## Overview: ##
Show More/Show Less of project descriptions needs to be implemented within My Projects

## PR Status: ##


## Related Jira tickets: ##

* [DES-2335](https://tacc-main.atlassian.net/browse/DES-2335)

## Summary of Changes: ##
Created a new _common folder at the components level
Moved the show-more temp & comp to the new _common folder
Now, Published and My Projects have access to this for the proj desc

## Testing Steps: ##
1. In My Projects, choose a project and create a looooong description. 
2. Use the Show More/Show Less buttons. Verify they work.
3. In Published, choose a project with a looooong description. 
4. Use the Show More/Show Less buttons. Verify they still work.

## UI 
<img width="1096" alt="Screen Shot 2024-01-03 at 11 25 03 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/d90852a1-31e5-4642-bb1a-c13f96ad6961">
<img width="1092" alt="Screen Shot 2024-01-03 at 11 24 54 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/8c21a542-499c-4a26-afce-b7d5c747487e">
Photos:

## Notes: ##
